base/workflow/R/start_model_runs.R

```diff
@@ -195,14 +209,17 @@ start_model_runs <- function(settings, write = TRUE, stop.on.error = TRUE) {
     }
     
     if (!is_local) {
-      for (run in run_list){
+      for (run in run_list){ #only re-copy run dirs that have launcher and job list
         if (run %in% job_modellauncher) {
           # copy launcher and joblist
-          PEcAn.remote::remote.copy.to(
-            host = settings$host,
-            src = file.path(settings$rundir, format(run, scientific = FALSE)),
-            dst = settings$host$rundir,
-            delete = TRUE)
+          PEcAn.utils::retry.func(
+            PEcAn.remote::remote.copy.to(
+              host = settings$host,
+              src = file.path(settings$rundir, format(run, scientific = FALSE)),
+              dst = settings$host$rundir,
+              delete = TRUE),
```

Comment:
Does this need to be `delete == TRUE`

Reply:
No, `=` is correct -- we're passing a named argument to `remote.copy.to`, not performing a logical comparison.